src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -130,13 +133,15 @@ private void sendAttack(long minWait, long bugTime, boolean normal) {
     }
 
     public double modifyRadius(double radius) {
-        if (target.health.hpPercent() < 0.25 && target.npcInfo.extra.has(NpcExtra.AGGRESSIVE_FOLLOW)) radius *= 0.75;
+        if (target.npcInfo.extra.has((NpcExtra.AGGRESSIVE_APPROACH)) && !closeRange) radius *= 0.55;
+        else if (target.health.hpPercent() < 0.25 && target.npcInfo.extra.has(NpcExtra.AGGRESSIVE_FOLLOW)) radius *= 0.75;
         if (target != hero.target || !hero.isAttacking(target) || castingAbility()) radius = Math.min(550, radius);
         else if (!target.locationInfo.isMoving() || target.health.hpPercent() < 0.25) radius = Math.min(600, radius);
+        radius += bar.findItemById("ability_zephyr_mmt").map(i -> i.quantity).orElse(0d) * 5;
 
-        return radius + bar.findItemById("ability_zephyr_mmt").map(i -> i.quantity).orElse(0d) * 5;
+        if (main.hero.locationInfo.distance(target) < radius) closeRange = true;
```

Comment:
If you put this at the very start of the function, it may be viable to see it happening
but in here, with the already reduced radius, it will likely never happen that you get closer